File: Baekjoon/S5/C#/S5_영화감독 숌.cs
https://www.acmicpc.net/problem/1436

using System;

namespace Beakjoon
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int num = 666;

            while (true)
            {
                if (n == 1)
                    break;

                num++;

                if (num.ToString().Contains("666"))
                    n--;
            }

            Console.Write(num);
        }
    }
}

File: Baekjoon/S5/JAVA/S5_영화감독 숌.java
//https://www.acmicpc.net/problem/1436

import java.io.*;
import java.util.*;

public class Main{
    public static void main(String args[]) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int num = 666;
        int cnt = 1;
        
        while(cnt != n){
            num++;
            
            if(Integer.toString(num).contains("666"))
                cnt++;
        }
        
        System.out.print(num);
    }
}